docs/logs/customizing-the-sdk/Program.cs

```diff
@@ -30,9 +30,7 @@ public static void Main()
             builder.AddOpenTelemetry(options =>
             {
                 options.IncludeScopes = true;
-                options.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(
-                    serviceName: "MyService",
-                    serviceVersion: "1.0.0"));
+                options.ConfigureResources(r => r.AddService(serviceName: "MyService", serviceVersion: "1.0.0"));
```

Comment:
We'll need to make sure we follow up with opentelemetry.io docs as well. Quite a few examples using `SetResourceBuilder` like: https://opentelemetry.io/docs/instrumentation/net/getting-started/#console-application